src/main/resources/static/js/AircraftStatus/AddAircraftEvent.js

```diff
@@ -156,21 +148,3 @@ function fetchReasons() {
         .catch(error => console.error('Error fetching reasons:', error));
 }
 
-function fetchCarriers() {
-    fetch('/getAllCarrier')
-        .then(response => response.json())
-        .then(data => {
-            const carrierSelects = document.getElementsByClassName('carrierSelect');
-
-            for (let i = 0; i < carrierSelects.length; i++) {
-                carrierSelects[i].innerHTML = '';
-                data.forEach(carrier => {
-                    const option = document.createElement('option');
-                    option.value = carrier.carrierId;
-                    option.textContent = carrier.carrierName;
-                    carrierSelects[i].appendChild(option);
-                });
-            }
-        })
-        .catch(error => console.error('Error fetching carriers:', error));
-}
```

Comment:
Sounds good.  This function wasn't needed anymore in AddAircraftEvent.js since the carrier is associated with the tail and  no longer part of the "Add a New Event" form.